editable-templates/mvp-reference-pages/table-component.html

```diff
@@ -0,0 +1,69 @@
+---
+title: Table component
+hide_breadcrumb: false
+date_modified: 2024-10-31
+layout: no-container
+---
+
+<div class="">
+	<div class="container">
+		<div class="">
+			<div class="row">
+				<div class="title col-lg-12 col-xs-12">
+					<h1 property="name" id="wb-cont" dir="ltr"> Table component
+					</h1>
+				</div>
+				<div class="table-responsive col-lg-12">
+					<table class="table table-bordered">
+						<tbody>
+							<tr>
+								<td class="text-center"><strong>R1C1</strong></td>
+								<td class="text-center"><strong>R1C2</strong></td>
+								<td class="text-center"><strong>R1C3 <sup>NOTE</sup></strong></td>
```

Comment:
This is not WCAG. 

When a `<strong>` is fully wrap with `<td>` in the tabular context means the element that we should see is `<th>`.